### PR TITLE
DVC Stages creation

### PR DESCRIPTION
**Pull request by ricardo8a**
#### What does this PR do?
creates dvc stages
data_prepare
featurize
elbow
train
#### How should this be manually tested?
run command
dvc repro
#### Any background context you want to provide?
verify exists csv.
#### Screenshots
the command create images in the doc directory

#### Type of change

- [x] New feature (non-breaking change which adds functionality)

#### Checklist

- [ ] I added the necessary documentation, if appropriate.
- [ ] I added tests to prove that my fix is effective, or my feature works.
- [ ] I reviewed existing Pull Requests before submitting mine.

### CODE REVIEW

File: data/raw/online_retail_raw.csv
Comment (trujillo9616) [inline data/raw/online_retail_raw.csv:1]: @ricardo8a Can we remove this file? We're tracking the data files with DVC so uploading the dataset to our repo is not a good practice.

**Review by trujillo9616 — CHANGES_REQUESTED** [review]
Is there a reason the `dvc.lock` file was not included? 🤔

File: data/raw/online_retail_raw.csv
Comment (ricardo8a) [reply data/raw/online_retail_raw.csv:1]: yes, I will remove that file

Comment (ricardo8a) [reply data/raw/online_retail_raw.csv:1]: I added because I cannot download from aws s3

**Review by trujillo9616 — APPROVED** [review]
🚀

**Pull request merged**